app/services/flow/flow_state_machine.rb

```diff
@@ -58,10 +116,15 @@ def issuer
 
     def fsm_initialize
       klass = self.class
+      flow = klass::FSM_SETTINGS[:flow]
+      @step_url = klass::FSM_SETTINGS[:step_url]
+      @final_url = klass::FSM_SETTINGS[:final_url]
+      @analytics_id = klass::FSM_SETTINGS[:analytics_id]
+      @view = klass::FSM_SETTINGS[:view]
       @name = klass.name.underscore.gsub('_controller', '')
-      klass::FSM_SETTINGS.each { |key, value| instance_variable_set("@#{key}", value) }
```

Comment:
one idea, I know I mentioned it in chat, but just posting for posterity:

In case anybody in the future expects the old behavior of *any* key in the hash getting turned into an instance variable, but that not happening because it's not listed here, one idea would be to delete from a copy as we go, and error if there are any left:


```ruby
fsm_settings = klass::FSM_SETTINGS.dup
flow = fsm_settings.delete(:flow)
@step_url = fsm_settings.delete(:step_url)
# ...
@view = fsm_settings.delete(:view)
raise "unknown FSM_SETTINGS keys: #{fsm_settings.keys.join(', ')}" unless fsm_settings.empty?
```